BitwardenShared/UI/Vault/Vault/VaultList/VaultListView.swift

```diff
@@ -188,7 +187,7 @@ private struct SearchableVaultListView: View {
     private var vaultFilterRow: some View {
         SearchVaultFilterRowView(
             hasDivider: false,
-            accessibilityID: "ActiveFilterRow",
+            accessibilityID: "ActiveFilterName",
```

Comment:
🤔 You're changing the `accessibilityID` here but you've removed the usage of this argument in the component [here](https://github.com/bitwarden/ios/pull/1252/files#diff-d56a00c4bc362738b3515c7a8b8f4668820b278a874635718b5d3bb7ea4b8d57L54)

Reply:
I forgot to remove this locator (it was replaced with the one I added on `SearchVaultFilterRowView`)
Thanks for noticing that 🙌